lms/djangoapps/notification_prefs/views.py

```diff
@@ -39,35 +45,22 @@ class UsernameCipher(object):
        decryption cipher
     5. base64url encode the result
     """
+    aes_block_size_bytes = int(AES.block_size / 8)
```

Comment:
Why not make this a module-level constant like in the other file?

Reply:
Because there wasn't an appropriate class scope in the other file, although it would read a little better if I made it module-scoped here too.  Changing.